Xamarin.Forms.ControlGallery.WindowsUniversal/MainPage.xaml.cs

```diff
@@ -127,6 +129,30 @@ void AddNativeControls(NestedNativeControlGalleryPage page)
 				}
 			);
 
+			{
+				var stackLayout = new StackLayout()
+				{
+					VerticalOptions = LayoutOptions.Start,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var label = new Label
+				{
+					Text = "Hello World!",
+					VerticalOptions = LayoutOptions.Center,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var bz56843TextBlock = new TextBlock
```

Comment:
Yeah, would have moved to a separate page but there was no precedence for that so... couldn't just copy paste. So I'll pull the "repro". It's not like we can't just look up the bug..